dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/ThreadPoolController.java

```diff
@@ -749,6 +775,7 @@ synchronized void resume() {
      * @return the data representing the throughput distribution for the
      *         specified number of active threads
      */
+    @Trivial
     ThroughputDistribution getThroughputDistribution(int activeThreads, boolean create) {
```

Comment:
The use of activeThreads in this method is confusing now that you have a instance scoped variable with the same name.  In reality activeThreads doesn't actually portray what this argument is.  numberOfThreads or something like that may be a better option.

Reply:
Agree, done.